src/SonarScanner.MSBuild.PreProcessor/Protobuf/AnalysisCacheMsg.cs

```diff
@@ -0,0 +1,224 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: AnalysisCacheMsg.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021, 8981
```

Comment:
This file should not be check into the repo, but autogenerated as part of the build from the *.proto file.

We need to update `.gitignore`

Reply:
For reference: https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Protobuf/.gitignore